packages/ui/src/molecules/RadioGroup/RadioOption.tsx

```diff
@@ -0,0 +1,49 @@
+import type { InputHTMLAttributes } from 'react'
```

Comment:
This component is a Label, basically, wrapper with a Radio inside. IMO, this component isn't necessary.

Reply:
Also, the user can mount it anyways, e.g:

```tsx
<Label>...</Label>
<Radio />
```
or
```tsx
<Label>
  <Radio />
</Label>
```
or
```tsx
<div id="foo">...</Label>
<Radio aria-labelledby="foo" />
```

Reply:
Yes, this component is a Label with a Radio inside, but using a context that helps to link the RadioOptions, style and maintain the accessibility, I got a base in [Chakra UI Implementation
](https://github.com/chakra-ui/chakra-ui/blob/main/packages/radio/src/radio.tsx)
I think If the user wants to do this in another form, he can use the atoms and customize it